src/endpoints/refreshEndpoint.ts

```diff
@@ -34,7 +34,9 @@ export const refresh = (
     const id = hpkeKeyIdMap.size + 1;
 
     // Generate HPKE key pair with a six digit id
-    const keyItem = KeyGeneration.generateKeyItem(100000 + id);
+    // since OHTTP is limited to 2 char ids, we can only have ids from 10 to 99
+    // So the current logic is to have ids rotate from 10 to 99
+    const keyItem = KeyGeneration.generateKeyItem(id % 90 + 10);
```

Comment:
I believed that our id was converted to one hex byte in BA services. Is this assumption wrong?

Reply:
yes, it turned out that BA service need a 2 char string but in as a decimal rather than hex